docs/config.toml

```diff
@@ -28,6 +28,13 @@ menushortcutsnewtab = true # set true to open shortcuts links to a new tab/windo
 enableGitInfo = true
 operatorVersion = "5.0.3"
 centosBase = "centos8"
+imageCrunchyPostgres = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0"
+imageCrunchyPGBackrest = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:centos8-2.36-0"
```

Comment:
⛏️ we're not 100% consistent, but it seems like we have more refs to PGBackRest than PGBackrest